### DISCUSSION TITLE
What is the Root Password for Preinstalled Desktop Image For OPI 5 Plus

### DISCUSSION THREAD

**Discussion by JFLim1** [Joshua-Riek/ubuntu-rockchip#425] [Q&A]
Device: Orange Pi 5 Plus
Ubuntu Image Installed: [ubuntu-22.04.3-preinstalled-desktop-arm64-orangepi-5-plus.img.xz](https://github.com/Joshua-Riek/ubuntu-rockchip/releases/download/v1.28/ubuntu-22.04.3-preinstalled-desktop-arm64-orangepi-5-plus.img.xz)

I tried to login as root or on terminal command line "su" but do not have the "root password".  During the first boot up of the image there was no requirement to enter the "root password".  

Checking what is the "root password"?

**Comment by Joshua-Riek** [comment]
Because of the OEM installer, the root password is not set, so you can not log in.

There is an oem user my default, but its password is random and thrown away for security. 

You need to follow the on-screen setup wizard to setup your system and create your account, this will also set the root password. Then you can login from a console.

**Reply by JFLim1** [reply]
Thanks. Noted.  The next time I flash the image will look out for setting "root" account.  Must have missed the last round.

**Comment by JFLim1** [comment]
> You need to follow the on-screen setup wizard to setup your system and create your account, this will also set the root password. Then you can login from a console.


Hi @Joshua-Riek,
Just flash Ubuntu-22.04.3 preinstalled desktop v1.29 on SD Card. Followed the on-screen setup wizard but there is NO option to setup "root" account only user account.

![image](https://github.com/Joshua-Riek/ubuntu-rockchip/assets/75384411/c9046c63-fc6c-49ff-b065-5772c1a2ed12)

Does this means thee is no way to login as "root"?  For example, the command "su" could be activated as the user password will not clear the "authentication"

```
jfl@jfl-opi5plus:~$ su
Password: 
su: Authentication failure
jfl@jfl-opi5plus:~$ 
```

**Reply by Joshua-Riek** [reply]
The root password is the same as your created user account password. 

Use `sudo su` to login as root.

**Comment by JFLim1** [comment]
> The root password is the same as your created user account password.
> 
> Use `sudo su` to login as root.

Thank you.  "sudo su" works

```
jfl@jfl-opi5plus:~$ sudo su
[sudo] password for jfl: 
root@jfl-opi5plus:/home/jfl#
```
